playbooks/support/responding.md

```diff
@@ -11,6 +11,14 @@ respond to these issues.
   These automated system alerts _may_ be the first sign of a problem but don't _necessarily_ warrant treatment as
   immediate incidents. E.g., load-balancers may route around failing EC2 instances or auto-scaling may bring up
   capacity in response to load.
+- Monitor automated uptime incidents created by OpsGenie.
+  - For high-criticality systems, OpsGenie will open incidents if
+    [Pingdom](https://my.pingdom.com/app/newchecks/checks) uptime checks are failing. Alerts will automatically
+    close if resolved.
+  - All engineers should have the [OpsGenie mobile app](http://itunes.apple.com/us/app/opsgenie/id528590328?mt=8)
+    installed and be logged in.
+  - Select your preferences for push notifications, text messages, and/or phone calls for alerting in your
```

Comment:
Perhaps recommend:
- Alert after 1 minute via push notification
- Alert after 5 minutes to SMS
- Alert after 20 minute via phone call